### ISSUE TITLE
Cannot download NAIP images on s3 using boto

### ISSUE THREAD

**Issue by andrewjavao** [trailbehind/DeepOSM#77]
I‘ve configurated s3cfg, and i can download naip image via s3cmd fine, but when running bin/create_training_data.py, it ends up 403 error when downloading naip images.

**Comment by andrewljohnson** [comment]
That sounds like when running s3cmd within the Docker/script, your credentials are not being used.

Did you export your AWS credentials per the README?

**Comment by andrewjavao** [comment]
I put the credentials into .s3cfg file right way inside the container, without setting docker container's environment parameter.
Maybe that is the problem, I'll try. I saw naip_images.py overwrites AWS credentials in .s3cfg with env-value in function configure_s3cmd(). Since I can download via s3cmd inside the container, so I modified the downloading code a little bit myself(just for debugging), I comment out boto client and os.system(s3cmd) to do the downloading. Though this works for me, but I'll try to setting up per the README, and make this issue clear.
